assets/main/sass/layout/_menu.scss

```diff
@@ -136,7 +136,7 @@
         border-radius: 11px;
         max-height: 60px;
         height: 100%;
-        font-size: 24px;
+        font-size: 23px !important;

```

Comment:
Why does it need to be `!important` - Is there another selector overriding this?

Reply:
There was a styling in #menu that did over-ride it. I removed it in commit 5d9aeae.